addons/l10n_in_edi/tests/test_edi_json.py

```diff
@@ -136,6 +136,10 @@ def setUpClass(cls, chart_template_ref="in"):
             "invoice_cash_rounding_id": rounding.id,
         })
         cls.invoice_cash_rounding.action_post()
+        cls.invoice_empty_product_hsn = cls.init_invoice("out_invoice", post=False, products=cls.product_a)
+        cls.invoice_empty_product_hsn.write({
+            "invoice_line_ids": [Command.update(line_id, {"l10n_in_hsn_code": ""}) for line_id in cls.invoice_empty_product_hsn.invoice_line_ids.ids]})
```

Comment:
```suggestion
        cls.invoice_empty_product_hsn.invoice_line_ids.l10n_in_hsn_code = ""
```
Sorry, didn't realise it sooner